src/utils/types.ts

```diff
@@ -430,6 +433,23 @@ interface StageDetails {
   description?: string
 }
 
+interface TemplateCategoryDetails {
+  categoryTitle: string
+  categoryIcon: string
+}
```

Comment:
Maybe doesn't need to be `categoryTitle` and `categoryIcon` just title and icon?

Reply:
7e81af9d3528a9842db91f456956ab1bfb6ece27

Although when it's destructured in dashboard it's i revert to categoryIcon and categoryTitle (to make it easier to see what the title and icon refers to in JSX